include/evmc/evmc.h

```diff
@@ -295,6 +295,9 @@ enum evmc_status_code
      */
     EVMC_WASM_TRAP = 16,
 
+    /** Balance of the caller does not have enough funds for value transfer. */
```

Comment:
I think the "balance of" is not necessary here: "The caller does not have enough funds for value transfer."

Reply:
Rephrased